utils/hct/hctstart.cmd

```diff
@@ -200,15 +208,15 @@ echo Please see the README.md instructions in the project root.
 exit /b 1
 endlocal
 
-:checkcmake
-setlocal
+:checkcmake 
 cmake --version | findstr 3.4.3 1>nul 2>nul
+if "0"=="%ERRORLEVEL%" exit /b 0
+cmake --version | findstr 3.7.2 1>nul 2>nul
+if "0"=="%ERRORLEVEL%" exit /b 0
+cmake --version | findstr /R 3.6.*MSVC 1>nul 2>nul
 if errorlevel 1 (
-  echo CMake 3.4.3 is the currently supported version - your installed cmake is not supported.
+  echo CMake 3.4.3 or 3.7.1 are the currently supported versions for VS 2015 and VS 2017 - your installed cmake is not supported.
```

Comment:
Is this actually needed anymore? I don't think we have problems with any specific versions of cmake.